geoportal/geoportailv3_geoportal/static-ngeo/less/layout.less

```diff
@@ -12,6 +12,10 @@ body {
   padding-top: @topbar-height;
 }
 
+.modal-backdrop {
```

Comment:
Should rebase and test with the fix: https://github.com/Geoportail-Luxembourg/geoportailv3/pull/3104/files before adding `style="z-index:1050"` to the modal